src/api/app/models/issue_tracker.rb

```
@@ -244,11 +244,15 @@ def parse_github_issues(js)
 
   def parse_github_issue(js, create = nil)
     issue = nil
-    if create
-      issue = Issue.find_or_create_by_name_and_tracker(js["number"].to_s, name)
-    else
-      issue = Issue.find_by_name_and_tracker(js["number"].to_s, name)
-      return if issue.nil?
+    begin
+      if create
+        issue = Issue.find_or_create_by_name_and_tracker(js["number"].to_s, name)
+      else
+        issue = Issue.find_by_name_and_tracker(js["number"].to_s, name)
+        return if issue.nil?
+      end
+    rescue TypeError => e
+      Airbrake.notify(e, { json_data: js })
```

This should still raising the exception? or we are going to fail silently?